src/components/App.tsx

```diff
@@ -19,13 +22,37 @@ export default function App() {
     initAuth();
   }, [setCurrentUser]);
 
+  const [opened, { toggle }] = useDisclosure();
+
   return (
-    <div id="app-container">
-      <Header />
-      <Switch>
-        <Route path="/" component={Landing} />
-        <Route component={PageNotFound} />
-      </Switch>
-    </div>
+    <MantineProvider>
+      <AppShell
+        header={{ height: 60 }}
```

Comment:
It feels kinda weird having a few styling things like the header height here instead of with the CSS but it wasn't obvious how to do this otherwise, and "kinda weird" is fine.